lib/galaxy/model/orm/engine_factory.py

```diff
@@ -25,16 +27,6 @@ def log_request_query_counts(req_id):
 
 
 def build_engine(url, engine_options, database_query_profiling_proxy=False, trace_logger=None, slow_query_log_threshold=0, thread_local_log=None, log_query_counts=False):
-    # Should we use the logging proxy?
-    if database_query_profiling_proxy:
```

Comment:
Shouldn't we drop the option or implement a replacement?

Reply:
I think a replacement would look like the slow query logger below, the `ConnectionProxy` that logging_connection_proxy uses has been removed (https://docs.sqlalchemy.org/en/14/changelog/changelog_14.html?highlight=connectionproxy#change-6daf2f59ac2438ef9c8213e9ebeac157)

Reply:
In fact setting `slow_query_log_threshold`  to 0 should be almost equivalent.

Reply:
I've added a replacement in https://github.com/galaxyproject/galaxy/pull/10856/commits/23852b22d771e928e169c46858c9b79bab0538c7